app/util/ContentUpgrade.scala

```diff
@@ -28,7 +31,7 @@ object ContentUpgrade {
   def upgradeResponse(json: JValue) = {
     json \ "response" match {
       case jsObject: JObject =>
-        JObject("response" -> (jsObject update JObject(ContentFields flatMap { field =>
+        JObject("response" -> (jsObject ~ JObject(ContentFields flatMap { field =>
```

Comment:
becasue `~` is in the native DSL, `update` was created by Rob as an implicit and had this comment

```
/** I really thought this method would exist in Json4s but I couldn't find it */
```